resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb

```diff
@@ -93,6 +111,11 @@ Requires:         chkconfig
 Requires:         aaa_base
 %endif
 
+<% if !EZBake::Config[:is_pe_build] -%>
+# Required to get trustanchors for java
+BuildRequires: %{_certs_package}
```

Comment:
Do we not need a certs package if this is foss? `ca-certs` and `ca-certs-mozilla` are both freely available, right? My gut is saying we'd want certs from `puppet-agent` if it's PE, and `ca-certs` or `ca-certs-mozilla` if it's foss